### DISCUSSION TITLE
It is proposed that the search results at "https://rubygems.org/search?query=" offer an additional "sorted by date/time" of last-released gem, on the right side of the current UI

### DISCUSSION THREAD

**Discussion by rubyFeedback** [rubygems/rubygems.org#3232] [Ideas]
It is here proposed to extend the search-result by showing a "last update"
entry. Here I refer to the right-hand side of the webpage at 
https://rubygems.org/search?query=.

Currently if you search for a gem, such as for gtk, as an example, via:

    https://rubygems.org/search?query=gtk

You get several results. But many of the projects listed
there are dead/inactive.

I would like to **propose** to:

**a**) add a date/time somewhere aka "last update", or something
similar to that.

and

**b**) to sort it by date automatically.

I would propose that a) is added at all times, but b) may
not be wanted by all users. For instance, some may prefer
default alphabetical sorting.

There is a "UPDATED LAST MONTH (9)", but the functionality
to sort **ALL** the found gems by DATE/TIME appears to be
missing (or at the least I could not see it).

I am not sure how to best integrate this, since there is already
functionality offered, like the "updated last xxx", but I think before
this is done anyway, it may be best to discuss  pro/cons how the 
above change may be useful for ruby users in general. And then
one could discuss implementation specifics or UI aspects to it.

I believe it is a useful suggestion because right now I do not seem to
be able to search by date, so I have to click on each individual result
and check for the last release, to figure out how active the project
may be.

The "UPDATED LAST MONTH (9)" is not useless, of course, but I'd
rather want to see ALL the gems. Sometimes I look at older gems to
see if they are still useful, but I'd love a date-sorted list to keep track
of these gems more easily.

**Comment by simi** [comment]
We can add date of release next to the latest release version, that's good idea IMHO.
